packages/api/src/utils/getFullBlockHash.ts

```diff
@@ -0,0 +1,20 @@
+import { prisma } from "@blobscan/db";
+
+export async function getFullBlockHash(partialHash: string) {
```

Comment:
I think `autocompleteBlockHash` is more appropriate. What do you think?

Reply:
This comes from this [discussion](https://github.com/Blobscan/blobscan/pull/608#discussion_r1819902336)

I wanted the function name to indicate that we’re fetching the full block hash from an external source. How about `fetchCompleteBlockHash`?

> I think `autocompleteBlockHash` is more appropriate. What do you think?

Reply:
sounds too technical to me